server.js

```diff
@@ -94,6 +95,17 @@ app.get("/api/:collectionName/:id", async (req, res) => {
   }
 });
 
+app.get("/api/:collectionName/filter/:key", async (req, res) => {
+  const { collectionName, key } = req.params;
+  try {
+    const data = await getListByKey({ collectionName, key });
+    res.send(data);
+  } catch (e) {
+    console.error(e);
+    res.status(500).send("An unexpected tea pot exploded.");
```

Comment:
Consider using `418`
```suggestion
    res.status(418).send("An unexpected tea pot exploded.");
```

Reply:
Changed the error text to a normal one. 😉